website/docs/developer/research/research-localstack.md

```diff
@@ -59,6 +59,323 @@ Docker's localhost can be accessed on the parent machine at http://host.docker.i
+        localstack-container  | Starting mock EC2 service on http port 4566 ...
+        localstack-container  | Starting mock ES service on http port 4566 ...
+        localstack-container  | Starting mock Firehose service on http port 4566 ...
+        localstack-container  | Starting mock IAM service on http port 4566 ...
+        localstack-container  | Starting mock STS service on http port 4566 ...
+        localstack-container  | Starting mock Kinesis service on http port 4566 ...
+        localstack-container  | Initializing DynamoDB Local with the following configuration:
+        localstack-container  | Port:   52091
+        localstack-container  | InMemory:       false
+        localstack-container  | DbPath: /tmp/localstack/data/dynamodb
+        localstack-container  | SharedDb:       false
+        localstack-container  | shouldDelayTransientStatuses:   false
+        localstack-container  | CorsParams:     *
+        localstack-container  | Starting mock KMS service on http port 4566 ...
+        localstack-container  |
+        localstack-container  | INFO[2021-06-16 16:40:34.915] Local KMS 3.9.1 (4e5eb289efb23f9059ae4b1f9896bad6c9a499f2)
+        localstack-container  | INFO[2021-06-16 16:40:34.936] No file found at path /init/seed.yaml; skipping seeding.
+        localstack-container  | INFO[2021-06-16 16:40:34.937] Data will be stored in /tmp/localstack/data
+        localstack-container  | INFO[2021-06-16 16:40:34.937] Local KMS started on 0.0.0.0:42519
+        localstack-container  | Listening at http://:::40789
+        localstack-container  | 2021-06-16 16:40:34,603:API:  * Running on all addresses.
+        localstack-container  |    WARNING: This is a development server. Do not use it in a production deployment.
+        localstack-container  | 2021-06-16 16:40:34,603:API:  * Running on http://172.21.0.2:39023/ (Press CTRL+C to quit)
+        localstack-container  | Starting mock Lambda service on http port 4566 ...
+        localstack-container  | Starting mock CloudWatch Logs service on http port 4566 ...
+        localstack-container  | Starting mock Redshift service on http port 4566 ...
+        localstack-container  | Starting mock Route53 service on http port 4566 ...
+
+Localstack is now running in docker. To verify that the services are running, check the link http://localhost:4566/ should show a status of "running".
+Open the docker desktop and click on the container's CLI to open the container.
+Install the AWS CLI to run CLI commands in the docker container.
+
+    pip install --trusted-host pypi.org --trusted-host files.pythonhosted.org awscli-local
+
+This prevents the need for specifying --endpoint-url everytime aws cli command is run. The container is now ready to test AWS services locally.
+
+### Examples of AWS services deployed using localstack
+Some examples of CLI commands and their corresponding outputs are shown below.
+#### API gateway
```

Comment:
You may want to how via an info admonition what `awslocal` is an alias for so that a new user can setup the alias quickly.